apiserver/deploy/base/insecure/kustomization.yaml

```diff
@@ -6,9 +6,10 @@ namespace: ray-system
 resources:
 - apiserver.yaml
 
-commonLabels:
-  app.kubernetes.io/name: kuberay
-  app.kubernetes.io/component: kuberay-apiserver
+labels:
+  - pairs:
+      app.kubernetes.io/name: kuberay
+      app.kubernetes.io/part-of: kuberay-apiserver
```

Comment:
Why this is changed from `component` to `part-of`?

Reply:
thanks, changed to `component`